ajax/libs/starability/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "starability",
+  "filename": "starability-all.min.css",
+  "version": "2.0.0",
+  "description": "Starability.css is an accessible rating system with cute animations on top. No JavaScript included.",
+  "author": "Anna Migas <amjmigas@gmail.com>",
+  "keywords": [
+    "rating",
+    "accessibility",
+    "css",
+    "sass",
+    "scss",
+    "animations"
+  ],
+  "homepage": "https://lunarlogic.github.io/starability/",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/LunarLogic/starability.git"
+  },
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/LunarLogic/starability.git",
+    "fileMap": [
+      {
+        "basePath": "",
+        "files": [
+          "starability-*(css|minified)/starability.*+(css)",
```

Comment:
Let's use `starability-+(css|minified)/starability*`.
1. "+" is for requirement of at least one "css" or "minified".
2. `starability*` is simpler and flexible.